packages/create-expo-module/src/create-expo-module.ts

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env node
+
+import spawnAsync from '@expo/spawn-async';
+import { Command } from 'commander';
+import downloadTarball from 'download-tarball';
+import ejs from 'ejs';
+import fs from 'fs-extra';
+import path from 'path';
+import prompts from 'prompts';
+
+const packageJson = require('../package.json');
+
+// `yarn run` may change the current working dir, then we should use `INIT_CWD` env.
+const cwd = process.env.INIT_CWD || process.cwd();
```

Comment:
```suggestion
const CWD = process.env.INIT_CWD || process.cwd();
```
Later in the code I assumed invocation is lacking, so I suggest making constant variable be named using `UPPER_CASE` 🆙